<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Omics Platforms</title>

<script src="site_libs/header-attrs-2.10/header-attrs.js"></script>
<script src="site_libs/jquery-1.12.4/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/htmlwidgets-1.5.2/htmlwidgets.js"></script>
<link href="site_libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet" />
<script src="site_libs/datatables-binding-0.15/datatables.js"></script>
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.min.css" rel="stylesheet" />
<link href="site_libs/dt-core-1.10.20/css/jquery.dataTables.extra.css" rel="stylesheet" />
<script src="site_libs/dt-core-1.10.20/js/jquery.dataTables.min.js"></script>
<link href="site_libs/nouislider-7.0.10/jquery.nouislider.min.css" rel="stylesheet" />
<script src="site_libs/nouislider-7.0.10/jquery.nouislider.min.js"></script>
<link href="site_libs/selectize-0.12.0/selectize.bootstrap3.css" rel="stylesheet" />
<script src="site_libs/selectize-0.12.0/selectize.min.js"></script>
<link href="site_libs/crosstalk-1.1.0.1/css/crosstalk.css" rel="stylesheet" />
<script src="site_libs/crosstalk-1.1.0.1/js/crosstalk.min.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />
<link rel="shortcut icon" href="resources/images/favicon_ITCR.ico" />

 <!--- 11/8/22 Awan: added the favicon_ITCR.ico file to the resources/image folder and included the favicon to the website. --->

 <!--- go to https://favicon.io/favicon-converter/ to upload an image to make a new favicon.io. You will need to replace the current favicon.io image with the one in the downloaded directory from the website. The current image is in the resources/images/ directory --->

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html"><img id="logo" style="width: 75px;" src="ITCRLogo.png" /></a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    <span class="fa fa-desktop"></span>
     
    Computing Resources
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="platformsOverview.html">
        <span class="fa fa-search-plus"></span>
         
        Platforms overview
      </a>
    </li>
    <li>
      <a href="multiTable.html">
        <span class="fa fa-stream"></span>
         
        Multi-data type platforms
      </a>
    </li>
    <li>
      <a href="imagingTable.html">
        <span class="fa fa-camera-retro"></span>
         
        Imaging platforms
      </a>
    </li>
    <li>
      <a href="omicsTable.html">
        <span class="fa fa-dna"></span>
         
        Omics platforms
      </a>
    </li>
    <li>
      <a href="clinicalTable.html">
        <span class="fa fa-heartbeat"></span>
         
        Clinical platforms
      </a>
    </li>
    <li>
      <a href="allToolsTables.html">
        <span class="fa fa-th"></span>
         
        All platforms
      </a>
    </li>
  </ul>
</li>
<li>
  <a href="dataResourceTable.html">
    <span class="fa fa-database"></span>
     
    Data Resources
  </a>
</li>
<li>
  <a href="contact.html">
    <span class="fa fa-envelope"></span>
     
    Contact
  </a>
</li>
<li>
  <a href="feedback.html">
    <span class="fa fa-comments"></span>
     
    Feedback Form
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore"><strong>Omics Platforms</strong></h1>

</div>


<div id="overview" class="section level3">
<h3><strong>Overview</strong></h3>
<p>The following table contains a list of “omics” informatics platforms funded by ITCR grants. “Omics” platforms are those which work with genomic or proteomic data types, such as RNA-Seq data. These platforms allow access to interactively explore and analyze large-scale cancer data, as well as create and share reproducible workspaces and results. The goals of these platforms are to facilitate scientific collaboration and to better empower the research community for “omic” research.</p>
<strong>Table keys:</strong>
<div title="Table keys">
<p><img src="resources/images/Hyperlink2.png"  height="30"> </img> = Main Link, <img src="resources/images/Documentation2.png" height="30"> </img> = Documentation, <img src="resources/images/Publication2.png" height="30"> </img> = Publications,   <img src="ITCRLogo.png" height="25"> </img> = ITCR Funded</p>
</div>
<div id="htmlwidget-5187709674451a00c711" style="width:100%;height:auto;" class="datatables html-widget"></div>
<script type="application/json" data-for="htmlwidget-5187709674451a00c711">{"x":{"filter":"top","filterHTML":"<tr>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n  <td data-type=\"character\" style=\"vertical-align: top;\">\n    <div class=\"form-group has-feedback\" style=\"margin-bottom: auto;\">\n      <input type=\"search\" placeholder=\"All\" class=\"form-control\" style=\"width: 100%;\"/>\n      <span class=\"glyphicon glyphicon-remove-circle form-control-feedback\"><\/span>\n    <\/div>\n  <\/td>\n<\/tr>","data":[["<b>GenePattern<\/b>:<br><\/br><a href=\"https://www.genepattern.org/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://notebook.genepattern.org/docs/\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"https://www.genepattern.org/papers\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a><br><\/br><img src=\"ITCRLogo.png\" height=\"40\"><\/img>","<b>QIIME 2<\/b>:<br><\/br><a href=\"https://qiime2.org/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://docs.qiime2.org/2022.2/\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"https://docs.qiime2.org/2022.2/citation/\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a><br><\/br><img src=\"ITCRLogo.png\" height=\"40\"><\/img>","<b>The BioMedical Evidence Graph (BMEG)<\/b>:<br><\/br><a href=\"https://bmeg.io/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://reporter.nih.gov/project-details/8840914\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"https://reporter.nih.gov/project-details/8840914\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a><br><\/br><img src=\"ITCRLogo.png\" height=\"40\"><\/img>","<b>P-Mart<\/b>:<br><\/br><a href=\"https://pmart.labworks.org/index.html\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"NA\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7032029/\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a><br><\/br><img src=\"ITCRLogo.png\" height=\"40\"><\/img>","<b>Cancer-Related Analysis of Variants Toolkit (CRAVAT)<\/b>:<br><\/br><a href=\"http://www.cravat.us/CRAVAT/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"http://www.cravat.us/CRAVAT/help.jsp#introduction\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"NA\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a><br><\/br><img src=\"ITCRLogo.png\" height=\"40\"><\/img>","<b>The Cancer Dependency Map Portal<\/b>:<br><\/br><a href=\"https://depmap.org/portal/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://depmap.org/portal/documentation/\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"https://depmap.org/portal/publications/\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a><br><\/br><img src=\"ITCRLogo.png\" height=\"40\"><\/img>","<b>cBioPortal for Cancer Genomics<\/b>:<br><\/br><a href=\"http://www.cbioportal.org/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://docs.cbioportal.org/5.2-datasets/downloads#introduction\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"https://www.cbioportal.org/tutorials\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a><br><\/br><img src=\"ITCRLogo.png\" height=\"40\"><\/img>","<b>Galaxy-P multi-omics<\/b>:<br><\/br><a href=\"http://galaxyp.org/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://training.galaxyproject.org/training-material/\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"http://galaxyp.org/publications/\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a><br><\/br><img src=\"ITCRLogo.png\" height=\"40\"><\/img>","<b>modelhub.ai<\/b>:<br><\/br><a href=\"http://modelhub.ai/#\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://modelhub.readthedocs.io/en/latest/\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"NA\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a><br><\/br><img src=\"ITCRLogo.png\" height=\"40\"><\/img>","<b>Overture<\/b>:<br><\/br><a href=\"https://www.overture.bio/about-us/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://www.overture.bio/documentation/\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"https://github.com/overture-stack/billing/blob/develop/README.md\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a><br><\/br><img src=\"ITCRLogo.png\" height=\"40\"><\/img>","<b>WebMeV<\/b>:<br><\/br><a href=\"https://www.hsph.harvard.edu/qbrc/cloud-services/webmev/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://web-mev.github.io/\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"https://webmev.tm4.org/#/about\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a><br><\/br><img src=\"ITCRLogo.png\" height=\"40\"><\/img>","<b>AnVIL<\/b>:<br><\/br><a href=\"https://anvilproject.org/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://help.anvilproject.org/\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"https://anvilproject.org/overview/publications\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a>","<b>Cancer Genome Collaboratory<\/b>:<br><\/br><a href=\"https://cancercollaboratory.org/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://cancercollaboratory.org/support/user-guides\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"https://cancercollaboratory.org/publications-presentations\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a>","<b>Genomic Data Commons Data Portal<\/b>:<br><\/br><a href=\"https://portal.gdc.cancer.gov/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://docs.gdc.cancer.gov/\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"https://gdc.cancer.gov/about-data/publications\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a>","<b>BaseSpace<\/b>:<br><\/br><a href=\"https://login.illumina.com/platform-services-manager/?rURL=https://basespace.illumina.com&clientId=basespace&clientVars=aHR0cHM6Ly9iYXNlc3BhY2UuaWxsdW1pbmEuY29tL2Rhc2hib2FyZA&redirectMethod=GET#/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://support.illumina.com/sequencing/sequencing_software/basespace/documentation.html\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"https://www.illumina.com/science/genomics-research/publications.html\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a>","<b>Cancer Genomics Cloud<\/b>:<br><\/br><a href=\"https://www.cancergenomicscloud.org/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://docs.cancergenomicscloud.org/docs\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"https://www.cancergenomicscloud.org/collaborate\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a>","<b>ISB Cancer Gateway in the Cloud<\/b>:<br><\/br><a href=\"https://isb-cgc.appspot.com/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"https://isb-cgc.appspot.com/citations/\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a>","<b>FireCloud<\/b>:<br><\/br><a href=\"https://firecloud.terra.bio/#\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Main Link\"> <\/div><img src=\"resources/images/Hyperlink2.png\"  height=\"30\"> <\/img><\/a><a href=\"https://support.terra.bio/hc/en-us/categories/360001399872\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Documentation\"> <\/div><img src=\"resources/images/Documentation2.png\" height=\"30\"> <\/img><\/a><a href=\"NA\"style=\"color: #be3b2a\"target=\"_blank\"<div title=\"Publications\"> <\/div><img src=\"resources/images/Publication2.png\" height=\"30\"> <\/img><\/a>"],["Genomics, Proteomics, Network Analysis","Microbiome Analysis","Genomics","Proteomics","Genomics","Genomics","Genomics","Genomics, Proteomics, Metabolomics","Clinical Modeling","Genomics","Genomics","Genomics","Genomics, Cloud Storage","Genomics","Genomics","Genomics, Proteomics,","Genomics, Proteomics,","Genomics, Proteomics"],["GenePattern is a genomic analysis platform that provides access to hundreds of tools for the analysis and visualization of multiple data types. A web-based interface provides easy access to these tools and allows the creation of multi-step analysis pipelines that enable reproducible in silico research. A new GenePattern Notebook environment allows users to combine GenePattern analyses with text, graphics, and code to create complete reproducible research narratives.","QIIME 2 is a microbiome bioinformatics platform that is widely used for investigation of the human microbiome, including studies investigating how our microbiota may impact carcinogenesis, how microbiota may impact the efficacy of cancer treatment, how cancer treatment may impact the microbiota, and associations between the microbiome and cancer prognosis. QIIME 2 was designed to ensure reproducible bioinformatics through its unqiue retrospective data provenance tracking system, and to facilitate accessibility to users with varying degrees of computational sophistication. QIIME 2 takes users from raw data through publication quality visualizations and statistics.","The BMEG is a Cancer Data integration Platform that utilizes methods collected from DREAM challenges and applied to large datasets, such as the TCGA, and makes them avalible for analysis using a high performance graph database","Web-based software platform to enable analysis and exploration of publicly available cancer proteomic datasets, such as those generated by the CPTAC program.","CRAVAT is a biologist-friendly platform for analysis of coding and non-coding cancer variants and has been extended to create the new Open-Custom Ranked Analysis of VAriants Toolkit, a configurable platform with community-developed annotation, visualization, and analysis tools. OpenCRAVAT provides an app store with 150+ tools, an interactive, richly detailed graphical interface, and is available for local or cloud-based installation. Webserver and Docker versions are also available for CRAVAT and OpenCRAVAT.","The DepMap Portal empowers researchers to make discoveries related to cancer vulnerabilities by providing open access to key datasets, analytical tools, and visualizations.","The cBioPortal for Cancer Genomics provides visualization, analysis and download of large-scale cancer genomics data sets.","A unified platform for integrative genomic-proteomic-metabolomic data analysis and informatics in cancer research.","Dissemination Platform for Deep Learning Algorithms","Overture is a platform designed to compile tools appropriate for genomic datasets.","WebMeV is a cloud-based platoform that facilitates the analysis, visualization, and stratification of genomic datasets.","AnVIL builds upon Galaxy and Terra to organize and store various genomics datasets. This platform has several catalogued bioinformatics tools, such as open-source R packages through its Bioconductor project.","The Cancer Genome Collaboratory is a cloud resource that allows researchers to conduct comprehensive analysis on the International Cancer Genome Consortium datasets.","The GDC Data Portal is a platform that contains several archives of data resources available for search and download by researchers. This platform also contains a data trasfer tool among other helpful modules for quick data analysis.","BaseSpace is a platform that works with Illumina sequencing data. This platform offers seamless integration with Illumina sequencing machines for easy data transfer and direct analysis.","The Cancer Genomics Cloud is a computing platform that allows researchers to analyze, store, and share their own data. CGC allows for user-uploaded data as well as large public and controlled cancer data sets, including genomic and imaging data.","The ISB-CRC Platform allows for users to browse data from the Genomic Data Commons and other large resources, including both public and controlled sequencing and imaging data. This platform offers several integrated tools as well as pipelines, workflows, and Notebooks for R and Python researchers.","FireCloud is a platform that offers computing resources and workspaces using Broad's tools and pipelines. Researchers can run large scale analyses with user data or TCGA controlled-access data."],["A web-based platform for genomic analysis; Allows for collaborative data, workflows, and pipelines","Open source; Free; Offers several tools for microbiome analysis","This platform offers data from several sources and provides a simple-to-use graphing system to reference and analyze data","This is a low-investment, beginner friendly platform for proteomics imaging and analysis","CRAVAT focuses on compiling tools for mutation interpretation; Can be used for other -omics resaerch as well","Scientists looking to map cancer dependancies should start here","This platform offers tools that allow researchers to specifically explore genes or pathways of interest; Open Source","Galaxy-P utilizes Galaxy as a framework to provide a platform that specifically works with spectrometry-based proteomics data analysis; This platform also offers wider proteomics and metabolomics research capabilities","Modelhub hosts deep learning models freely available for use by researchers.","Several collaborations with organizations like the Cancer Genome Collaboratory are linked through this platform; The data management system is a useful structure to build your own platform.","This platform focuses on large genomic data sets and NGS data; Free and open source","This platform offers methods to visualize genomic analyses that may be conducted using Galaxy and Terra.","Offers specific data access from the International Cancer Genome Consortium (ICGC).","This platform provides one of the most straightforward interfaces for genomic data access. Several tools are also built-in to allow for easy analysis.","Specifically focuses on providing several tools for Illumina sequencing data; Several tools available; 30 Day Free Trial","Educational resources through tutorial series, webinars, and guide documentation","Easy to browse data resources; Readily available tools, workflows, and Notebooks for collaborative efforts","Large-scale analysis capabilities, Collaborative workspace environment"],["Sequence, Microarray, Flow Cytometry, Liquid Chromatography/Mass Spectrometry","Sequence, Metadata Analysis","Sequence, Publication Text, Pathway Analysis,","Sequence, Proteome Datasets","Sequence, Variant Mapping, Sequence Structure Analysis","Sequence, Cellular Models, General Imaging, Modeling Data","Sequence, EMR Text,","Liquid Chromatography/Mass Spectrometry","Modeling Data, General Imgaging, Free Text","Sequence,","Sequence, Microarray","General Data","Sequence","Sequence","Sequence","Sequence","Sequence, Pathology Imaging, Radiographic Imaging","Sequence, Epigenetic Datasets"],["<a href=\"https://www.genepattern.org/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Free<\/a>","<a href=\"https://docs.qiime2.org/2021.11/install/native/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Free<\/a>","<a href=\"https://www.synapse.org/#!Synapse:syn312572/wiki/60903\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Free<\/a>","<a href=\"https://pmart.labworks.org/index.html\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Free<\/a>","<a href=\"http://www.cravat.us/CRAVAT/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Free<\/a>","<a href=\"https://depmap.org/portal/download/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Free<\/a>","<a href=\"http://www.cbioportal.org/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Free<\/a>","<a href=\"http://galaxyp.org/access-galaxy-p/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Free<\/a>","<a href=\"http://modelhub.ai/#\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Free<\/a>","<a href=\"https://github.com/overture-stack/billing/blob/develop/README.md\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Free<\/a>","<a href=\"https://web-mev.github.io/quickstart/#login\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Free<\/a>","<a href=\"https://anvilproject.org/learn/billing-setup/billing-concepts\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Paid<\/a>","<a href=\"https://billing.cancercollaboratory.org/login\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Paid<\/a>","<a href=\"https://portal.gdc.cancer.gov/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Free<\/a>","<a href=\"https://login.illumina.com/platform-services-manager/?rURL=https://basespace.illumina.com&clientId=basespace&clientVars=aHR0cHM6Ly9iYXNlc3BhY2UuaWxsdW1pbmEuY29tL2Rhc2hib2FyZA&redirectMethod=GET#/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Paid<\/a>","<a href=\"https://www.cancergenomicscloud.org/cloud-credits\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Paid/Credits<\/a>","<a href=\"https://isb-cancer-genomics-cloud.readthedocs.io/en/latest/sections/CostManagement.html\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Paid/Credits<\/a>","<a href=\"https://support.terra.bio/hc/en-us/articles/360026182251-How-to-set-up-billing-in-Terra\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Pricing Link\"> <\/div>Paid/Credits<\/a>"],["Curated Data Sources","Greengenes, Silva, UNITE, Qiita, SEPP, Curated Data Sources","<a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>TCGA<\/a>, MC3, GTEx, PharmacoDB, <a href=\"https://sites.broadinstitute.org/ccle/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>CCLE<\/a>, CTRP, GDSC, Ensembl, GO, MonDO, MSigDB, PFAM, PubChem, PubMed, VICCG2P, Pathway Commons, DGIDB","<a href=\"https://proteomics.cancer.gov/programs/cptac\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>CPTAC<\/a>, <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>TCGA<\/a>","No Data Provided","Curated Data Sources","<a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>TCGA<\/a>, <a href=\"https://ocg.cancer.gov/programs/target\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>TARGET<\/a>","Collaborative/User Data Sources","Curated Data Sources","<a href=\"https://dcc.icgc.org/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>ICGC<\/a>, GDC, Cancer Genome Collaboratory, <a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>TCGA<\/a>, <a href=\"https://ocg.cancer.gov/programs/target\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>TARGET<\/a>, Kids First Data Resource Center","No Data Provided","NHGRI, CCDG, CMG, eMERGE","<a href=\"https://dcc.icgc.org/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>ICGC<\/a>, PCAWG, DCC","<a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>TCGA<\/a>, <a href=\"https://ocg.cancer.gov/programs/target\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>TARGET<\/a>","Collaborative/User Data Sources","<a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>TCGA<\/a>, <a href=\"https://proteomics.cancer.gov/programs/cptac\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>CPTAC<\/a>, <a href=\"https://www.cancerimagingarchive.net/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>TCIA<\/a>, <a href=\"https://ocg.cancer.gov/programs/target\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>TARGET<\/a>, <a href=\"https://dcc.icgc.org/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>ICGC<\/a>, <a href=\"https://sites.broadinstitute.org/ccle/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>CCLE<\/a>, SGDP, PGP-UK","<a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>TCGA<\/a>, <a href=\"https://ocg.cancer.gov/programs/target\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>TARGET<\/a>, <a href=\"https://sites.broadinstitute.org/ccle/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>CCLE<\/a>, COSMIC","<a href=\"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>TCGA<\/a>, <a href=\"https://www.internationalgenome.org/\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>1000 Genomes<\/a>, AMP PD, Project Baseline, CCDG, CMG, ENCODE, Broad Dataset Workspace Library, <a href=\"https://ocg.cancer.gov/programs/target\"style=\"color: #be3b2a\" target=\"_blank\"<div title=\"Resource Link\"> <\/div>TARGET<\/a>, TOPMed, NeMO, HCA, GP2"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th>Platform<\/th>\n      <th>Subcategories<\/th>\n      <th>Summary<\/th>\n      <th>Unique Factor<\/th>\n      <th>Types of Data<\/th>\n      <th>Price<\/th>\n      <th>Data Provided<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"scrollX":true,"autoWidth":true,"pageLength":10,"scrollCollapse":true,"fillContainer":true,"order":[0,"asc"],"columnDefs":[{"className":"dt-center","targets":"_all"},{"width":"280px","targets":2},{"width":"130px","targets":3}],"initComplete":"function(settings, json) {\n$(this.api().table().header()).css({'backgroundColor': '#3f546f'});\n$(this.api().table().header()).css({'color': '#fff'});\n}","orderClasses":false,"orderCellsTop":true}},"evals":["options.initComplete"],"jsHooks":[]}</script>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
